client/dashboard/profile-wizard/steps/industry.js

```diff
@@ -80,13 +96,12 @@ class Industry extends Component {
 								/>
 							);
 						} ) }
+						{ error && <span class="woocommerce-profile-wizard__error">{ error }</span> }
```

Comment:
This is throwing a warning in the console for me `s/class/classname`

![image](https://user-images.githubusercontent.com/22080/61011711-92503b00-a330-11e9-8d16-e1b8b10c9b8c.png)

Reply:
Thanks!  Looks like my linter is on the fritz again.